addons/godot_xml/xml.gd

```diff
@@ -194,6 +199,9 @@ static func _attach_node_data(node: XMLNode, parser: XMLParser) -> void:
         # we therefore strip "blankets", resulting in only actual content slipping into .content
         node.content = parser.get_node_data().strip_edges().lstrip(" ").rstrip(" ")
 
+static func _attach_node_name(node: XMLNode, parser: XMLParser) -> void:
+    if node.content.is_empty():
+        node.content = parser.get_node_name().strip_edges().lstrip(" ").rstrip(" ")
```

Comment:
Yeah, this is very odd. I'm just happy it worked to solve my issue at first case, but prob it will start giving us problems in the future, so we should open an issue yeah.

Reply:
Also, feel free to open the issue yourself, since it was you that found this issue on Godot's documentation, so, your contribuition, not mine.